include/mbedtls/mbedtls_config.h

```diff
@@ -1632,6 +1632,22 @@
  */
 #define MBEDTLS_SSL_TLS1_3_DEFAULT_NEW_SESSION_TICKETS 1
 
+/**
+* \def MBEDTLS_SSL_EARLY_DATA
+*
+* Enable support for RFC 8446 TLS 1.3 early data.
+*
+* Requires: MBEDTLS_SSL_TLS1_3_KEY_EXCHANGE_MODE_PSK_ENABLED or
+*           MBEDTLS_SSL_TLS1_3_KEY_EXCHANGE_MODE_PSK_EPHEMERAL_ENABLED
+*
+* Comment this to disable support for early data.
```

Comment:
```suggestion
* Comment this to disable support for early data.  If MBEDTLS_SSL_PROTO_TLS1_3
* is not enabled, this option does not have any effect on the build.                                                  
```
and in build_info.h disable it automatically if MBEDTLS_SSL_PROTO_TLS1_3 is not defined
like MBEDTLS_SSL_TLS1_3_KEY_EXCHANGE_MODE_*_ENABLED. This should fix the CI failures.